storage/ndb/rest-server/rest-api-server/internal/config/init_defaults.go

```diff
@@ -93,5 +83,17 @@ func newWithDefaults() AllConfigs {
 			MaxBackups: 10,
 			MaxAge:     30,
 		},
+		Testing: Testing{
```

Comment:
Don't forget that Security also has TestingParameters. You can leave it there, just saying.